client/src/components/ThemeToggle/index.js

```diff
@@ -1,12 +1,17 @@
-import React, { useContext } from 'react';
+import React, { useContext, useEffect } from 'react';
 import { ThemeContext } from '../../ThemeContext.js';
 import { FiSun, FiMoon } from 'react-icons/fi';
 import styles from './Style.module.css';
+import storage from '../../utils/storage';
 
 const ThemeToggle = () => {
   const [darkMode, setDarkMode] = useContext(ThemeContext);
 
+  useEffect(() => {
+    setDarkMode(storage.get('theme'));
+  });
```

Comment:
Need to pass empty dependencies else this effect will be called after every render.

```suggestion
  useEffect(() => {
    setDarkMode(storage.get('theme'));
  }, []);
```